src/bootstrapper/singleInstance/InstanceLock.cpp

```diff
@@ -33,22 +34,18 @@ InstanceLock::~InstanceLock()
 
 bool InstanceLock::lock()
 {
-	if (create()) {
-		return true;
-	} else {
-		attachDetach();
-		return create();
-	}
+    if(this->create())
```

Comment:
Can you add braces around this return. 
Is "this->" here required, I think can be omitted.